File: sbillingFront/app/models/house.js
import DS from 'ember-data';

export default DS.Model.extend({
  addressString: DS.attr(),
  houseNumber: DS.attr(),
  street: DS.attr(),
  city: DS.attr(),
  name: DS.attr(),
  actuality: DS.attr(),

  workplace: DS.belongsTo('workplace'),

  employee: DS.hasMany('employee'),
  workshift: DS.hasMany('workshift'),
  room: DS.hasMany('room'),
  workshiftType: DS.hasMany('workshift-type'),
});

File: sBillingApi/Models/House.cs
using System;
using System.Collections.Generic;

// Code scaffolded by EF Core assumes nullable reference types (NRTs) are not used or disabled.
// If you have enabled NRTs for your project, then un-comment the following line:
// #nullable disable

namespace sBillingApi.Models
{
    public partial class House
    {
        public House()
        {
            Employee = new HashSet<Employee>();
            Room = new HashSet<Room>();
            Workshift = new HashSet<Workshift>();
            WorkshiftType = new HashSet<WorkshiftType>();
        }

        public Guid Id { get; set; }
        public string AddressString { get; set; }
        public string HouseNumber { get; set; }
        public string Street { get; set; }
        public string City { get; set; }
        public string Name { get; set; }
        public Guid? Workplace { get; set; }
        public bool? Actuality { get; set; }

        public virtual Workplace WorkplaceNavigation { get; set; }
        public virtual ICollection<Employee> Employee { get; set; }
        public virtual ICollection<Room> Room { get; set; }
        public virtual ICollection<Workshift> Workshift { get; set; }
        public virtual ICollection<WorkshiftType> WorkshiftType { get; set; }
    }
}
